cmd/multi-factor-auth/swagger/data.go

```diff
@@ -142,6 +142,28 @@ type _ struct {
 	}
 }
 
+// swagger:route POST /sign-message Guardian signMessageRequest
+// Sign transaction.
+// Signs the provided transaction with the provided guardian
```

Comment:
signs the provided message with the corresponding user guardian

Reply:
done